examples/basics/scene/mesh_texture.py

```diff
@@ -0,0 +1,46 @@
+import argparse
+
+import numpy as np
+from vispy import app, scene
+from vispy.io import imread, read_mesh
+from vispy.scene.visuals import Mesh
+from vispy.visuals.filters import TextureFilter
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--mesh', default='./data/spot.obj')
```

Comment:
Should be a little more clever about finding the data; maybe:
`data_path = os.path.join(os.path.dirname(__file__), 'data')`

Reply:
@asnt I would suggest to put the data into http://github.com/vispy/demo-data/. We can retrieve it from there. Have a look how this is done in https://github.com/vispy/vispy/blob/master/examples/basics/scene/volume.py

Reply:
@kmuehlbauer Thank you. I'll do this change.